apps/codeforafrica/src/components/NavBarNavList/NavBarNavList.js

```diff
@@ -0,0 +1,69 @@
+import { Link } from "@commons-ui/next";
+import SvgIcon from "@mui/material/SvgIcon";
+import PropTypes from "prop-types";
+import React from "react";
+
+import TwitterIcon from "@/codeforafrica/assets/icons/Type=twitter, Size=24, Color=CurrentColor.svg";
+import NavList from "@/codeforafrica/components/NavList";
+import NavListItem from "@/codeforafrica/components/NavListItem";
+
+const NavBarNavList = React.forwardRef(function NavBarNavList(props, ref) {
+  const { menu, direction, ...other } = props;
+
+  if (!menu?.length) {
+    return null;
+  }
+  return (
+    <NavList direction={direction} {...other} ref={ref}>
+      {menu.map((item) => (
+        <NavListItem key={item.label} sx={{ m: "20px" }}>
+          <Link
+            href={item.href}
+            color="inherit"
+            underline="none"
+            sx={{
+              typography: { xs: "h4", md: "body3" },
+              "&:hover, &:active, &:focus, &:focus-within": {
+                textDecoration: "none",
+                color: { xs: "inherit", md: "primary.main" },
+              },
+            }}
+          >
+            {item.label}
+          </Link>
+        </NavListItem>
+      ))}
+      <NavListItem sx={{ m: "20px", mr: 0 }}>
+        <Link
+          href="https://twitter.com/Code4Africa"
+          sx={{ color: { xs: "inherit" } }}
+        >
+          <SvgIcon
+            component={TwitterIcon}
+            sx={{
+              mt: direction === "column" ? 0 : 1,
+              fill: { xs: "text.secondary", md: "none !important" },
```

Comment:
```suggestion
              fill: { xs: "none" },
```
Even in mobile, we don't want `fill`:
![Mobile _390px](https://user-images.githubusercontent.com/1779590/169806969-60911811-0fe3-43a7-a187-16549b12cb11.png)